<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UpgradeWindow</class>
 <widget class="QMainWindow" name="UpgradeWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>502</width>
    <height>111</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QGridLayout" name="upgradesGridLayout">
      <item row="1" column="1">
       <widget class="QLabel" name="upgradeAutoClickerPriceLabel">
        <property name="text">
         <string>Цена: 1000 очков</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="upgradeClickValuePriceLabel">
        <property name="text">
         <string>Цена: 50 очков</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="upgradeClickValuePushButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Улучшить ценность клика</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="upgradeAutoClickerPushButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Улучшить автокликер</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="statsVerticalLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SizeConstraint::SetNoConstraint</enum>
      </property>
      <item>
       <widget class="QLabel" name="statsLabel">
        <property name="text">
         <string>Характеристики:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="currentClickValuerLevelLabel">
        <property name="text">
         <string>Очков за клик: 1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="currentAutoClickerLevelLabel">
        <property name="text">
         <string>Очков в секунду: 0</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>502</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="gameProgressMenu">
    <property name="title">
     <string>Игровой прогресс</string>
    </property>
    <addaction name="saveGameAction"/>
    <addaction name="loadGameAction"/>
   </widget>
   <addaction name="gameProgressMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="saveGameAction">
   <property name="text">
    <string>Сохранить игру</string>
   </property>
  </action>
  <action name="loadGameAction">
   <property name="text">
    <string>Загрузить игру</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
